test/parallel/test-whatwg-url-inspect.js

```diff
@@ -13,71 +12,56 @@ if (!common.hasIntl) {
 }
 
 // Tests below are not from WPT.
-const tests = require(path.join(common.fixturesDir, 'url-tests'));
-const additional_tests = require(
-  path.join(common.fixturesDir, 'url-tests-additional'));
+const url = new URL('https://username:password@host.name:8080/path/name/?que=ry#hash');
 
-const allTests = additional_tests.slice();
-for (const test of tests) {
-  if (test.failure || typeof test === 'string') continue;
-  allTests.push(test);
-}
-
-for (const test of allTests) {
-  const url = test.url ? new URL(test.url) : new URL(test.input, test.base);
-
-  for (const showHidden of [true, false]) {
-    const res = util.inspect(url, {
-      showHidden
-    });
-
-    const lines = res.split('\n');
+assert.strictEqual(
+  util.inspect(url),
+  `URL {
+  href: 'https://username:password@host.name:8080/path/name/?que=ry#hash',
+  origin: 'https://host.name:8080',
+  protocol: 'https:',
+  username: 'username',
+  password: '--------',
+  host: 'host.name:8080',
+  hostname: 'host.name',
+  port: '8080',
+  pathname: '/path/name/',
+  search: '?que=ry',
+  searchParams: URLSearchParams { 'que' => 'ry' },
+  hash: '#hash' }`);
 
-    const firstLine = lines[0];
-    assert.strictEqual(firstLine, 'URL {');
+assert.strictEqual(
+  util.inspect(url, { showHidden: true }),
+  `URL {
+  href: 'https://username:password@host.name:8080/path/name/?que=ry#hash',
+  origin: 'https://host.name:8080',
+  protocol: 'https:',
+  username: 'username',
+  password: 'password',
+  host: 'host.name:8080',
+  hostname: 'host.name',
+  port: '8080',
+  pathname: '/path/name/',
+  search: '?que=ry',
+  searchParams: URLSearchParams { 'que' => 'ry' },
```

Comment:
That is to be consistent with the inspection output of `Map`, plus it's weird to write

```js
{ query: 'value1',
  query: 'value2' }
```

with object literal syntax when it doesn't really work.